src/langs/en.json

```
@@ -111,5 +111,8 @@
   "EXPORT_INFORMATIONS_DESCRIPTION": "You can download a copy of your data.",
   "EXPORT_INFORMATIONS_BUTTON_TEXT": "Download your data",
   "EXPORT_SUCCESS_MESSAGE": "You have successfully requested your data. You will receive an e-mail with your data in a few minutes.",
-  "EXPORT_ERROR_MESSAGE": "An unexpected error occurred"
+  "EXPORT_ERROR_MESSAGE": "An unexpected error occurred",
+  "NOT_FOUND_PAGE_TEXT": "404 Page not found",
```

Actually I don't know if we should show the 404 error code, as this is not really something that users should see no ?
```suggestion
  "NOT_FOUND_PAGE_TEXT": "Page not found",
```